bot/exts/nominations.py

```diff
@@ -0,0 +1,81 @@
+import re
+
+import discord
+from discord.ext import commands
+
+from bot import constants, logger
+from bot.bot import ThreadBot
+from bot.utils.helpers import get_thread_from_message_id
+
+NOMINATION_MESSAGE_REGEX = re.compile(
+    r"<@!?\d+> \((.+)#\d{4}\) for Helper!\n\n\*\*Nominated by:\*\*",
+    re.MULTILINE
+)
+
+# When nominations are posted manually, the Discord message box converts +1/-1 to the thumbs emojis
+NOMINATION_ENDING_TEXT = "react 👍 for approval, or 👎 for disapproval*."
+
+
+class Nominations(commands.Cog):
+    """Cog for getting information about PyPi packages."""
+
+    def __init__(self, bot: ThreadBot):
+        self.bot = bot
+        # Stores the name of the member recently posted nomination as,
+        # nomiantion messages can be split into multiple messages.
+        self.nominated_member_name: int = None
+
+        if constants.DEBUG_MODE:
+            self.archive_time = constants.ThreadArchiveTimes.DAY.value
+        else:
+            self.archive_time = constants.ThreadArchiveTimes.WEEK.value
```

Comment:
Some suggestions, not sure what's best really..
```suggestion
        self.archive_time = constants.ThreadArchiveTimes.DAY.value if constants.DEBUG_MODE else constants.ThreadArchiveTimes.WEEK.value
```
```suggestion
        seconds = constants.ThreadArchiveTimes.DAY.value if constants.DEBUG_MODE else constants.ThreadArchiveTimes.WEEK.value
        self.archive_time = seconds
```
```suggestion
        archive_time = constants.ThreadArchiveTimes.DAY if constants.DEBUG_MODE else constants.ThreadArchiveTimes.WEEK
        self.archive_time = archive_time.value
```
I guess it may simply be best the way it's done

Reply:
I ended up with what I've got since I didn't want to deal with a very long conditional expression (which broke the line limit)